Source/Core/Core/BootManager.cpp

```diff
@@ -279,6 +279,7 @@ bool BootCore(std::unique_ptr<BootParameters> boot, const WindowSystemInfo& wsi)
       config_cache.bSetVolume = true;
     dsp_section->Get("EnableJIT", &StartUp.m_DSPEnableJIT, StartUp.m_DSPEnableJIT);
     dsp_section->Get("Backend", &StartUp.sBackend, StartUp.sBackend);
+    VideoBackendBase::ActivateBackend(Config::Get(Config::MAIN_GFX_BACKEND));
```

Comment:
The root issue is that the new graphics backend gets initialized too late after the WindowSystemInfo has been prepared for the *original* video backend (and not for the new backend).

https://github.com/dolphin-emu/dolphin/blob/4cd1567308ab921d0e37a83672ad91fa4932097f/Source/Core/Core/Core.cpp#L246-L253

Could you move this line to right before the above code snippet (before "// Issue any API calls...")? This will make it clearer that the manual ActivateBackend isn't redundant.

Reply:
Just pushed with these changes!